src/docfx.website.themes/pdf.default/cover.html.primary.tmpl

```diff
@@ -0,0 +1,15 @@
+{{!Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.}}
+{{!master(layout/_master.tmpl)}}
+<style>
+.cover-title {
+    height: 200px;
+    width: 400px;
+    position: fixed;
+    text-align: center;
+    top: 50%;
+    left: 50%;
+    margin-top: -100px;
+    margin-left: -200px;
+}
+</style>
+<div class="cover-title">{{{rawTitle}}}</div>
```

Comment:
We can also add `{{{conceptuall}}}` below. Although it could not be of the ideal format in cover page, users should be happier than finding something missing unexpectedly in output cover.